spec/controllers/base_indexer/items_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+describe BaseIndexer::ItemsController, type: :controller do
+  let(:my_instance) { instance_double(BaseIndexer::MainIndexerEngine) }
+  before do
+    allow(BaseIndexer::MainIndexerEngine).to receive(:new).and_return(my_instance)
+  end
+  describe 'PATCH/PUT update' do
+    it 'creates an indexing job' do
+      expect(my_instance).to receive(:index).with('bb1111cc2222', 'SEARCHWORKS' => true)
+      patch :update, druid: 'druid:bb1111cc2222', subtarget: 'SEARCHWORKS', use_route: :base_indexer
+      expect(response.status).to eq 200
+    end
+    it 'when something bad happens return a 202' do
+      expect(my_instance).to receive(:index).with('bb1111cc2222', 'SEARCHWORKS' => true).and_raise(StandardError)
+      patch :update, druid: 'druid:bb1111cc2222', subtarget: 'SEARCHWORKS', use_route: :base_indexer
+      expect(response.status).to eq 202
```

Comment:
is 202 the right code to return on a failure? not sure what rails recommends but a 5xx seems more correct -- like  `500 `:internal_server_error` or even a 503 `:service_unavailable`. looks like an API change but seems better to use a 5xx server error code. maybe this is a seperate issue though this this PR doesn't change this behavior...